pkg/console/install_panels.go

```diff
@@ -1293,7 +1345,13 @@ func addNetworkPanel(c *Console) error {
 			if err != nil {
 				return "", err
 			}
-			mgmtNetwork.MTU, err = strconv.Atoi(mtu)
+
+			if mtu != "" {
+				mgmtNetwork.MTU, err = strconv.Atoi(mtu)
+			} else {
+				mgmtNetwork.MTU = 0
```

Comment:
should leave it unset instead of `0`

Reply:
Please leave it unset and let it as the default value.